databricks/sdk/core.py

```diff
@@ -903,6 +925,7 @@ def __init__(self,
                  status: str = None,
                  scimType: str = None,
                  error: str = None,
+                 details: List[ErrorDetail] = [],
```

Comment:
On second look, this type hint is incorrect because the constructor expects a `Dict[str, any]`.